packages/web3-providers/src/alchemy/constants.ts

```diff
@@ -0,0 +1,31 @@
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { ChainId as ChainId_EVM } from '@masknet/web3-shared-evm'
+import { ChainId as ChainId_FLOW } from '@masknet/web3-shared-flow'
+export const Alchemy_EVM_NetworkMap = {
+    network: NetworkPluginID.PLUGIN_EVM,
+    chains: [
+        {
+            chainId: ChainId_EVM.Mainnet,
+            API_KEY: '3TJz6QYDHCj0ZhCdGvc5IC6EtMMMTKG1',
```

Comment:
Should we put this api key in r2d2.to proxy?

Reply:
+1